csv/result.go

```diff
@@ -71,6 +71,10 @@ func NewResultDecoder(c ResultDecoderConfig) *ResultDecoder {
 
 // ResultDecoderConfig are options that can be specified on the ResultDecoder.
 type ResultDecoderConfig struct {
+	// NoAnnotations indicates that the CSV data will not have annotations rows.
+	// Without annotations the decoder will assume every column is of type string and
+	// that all data is in a single table in a single result.
+	NoAnnotations bool
```

Comment:
What will the column names be though? I guess I'll see as I read on.

Reply:
Ah, ok. The column labels are not actually an annotation. They are still required.